src/gh/components/DF_joint_segmentator/metadata.json

```diff
@@ -0,0 +1,92 @@
+{
+    "name": "Extract Joints",
+    "nickname": "Extract Joints",
+    "category": "diffCheck",
+    "subcategory": "Segmentation",
+    "description": "Extracts the joints from a point cloud.",
+    "exposure": 4,
+    "instanceGuid": "fd804928-bf11-43eb-a29f-8afabefa660d",
```

Comment:
is just double-checkin 🐔 : is it unique in respect to the others?

Reply:
the guid is unique